modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,34 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
+$ curl https://helloworld-go.default.example.com
+Hello World: Go Sample v1!
+----
+
+== Self signed certificates
+
+If you receive an error relating to a self-signed certificate in the certificate chain, you can add the ``--insecure` flag to the curl command to ignore the error.
+
+[IMPORTANT]
+====
+Self-signed certificates must not be used in a production deployment. This method is only for testing purposes.
+====
+
+.Example
+----
+$ curl https://helloworld-go.default.example.com --insecure
+Hello World: Go Sample v1!
+----
+
+== Certificate authority signed certificates
```

Comment:
Since trying to hyphenate this might be awkward, I might reword to "Certificates signed by a certificate authority"

Reply:
These also are kind of mini procedures but don't follow the procedure module template. I'd consider updating to do so / use a gerund heading, etc. Or consider incorporating into the above procedure steps.